src/olympia/activity/tests/test_models.py

```diff
@@ -230,6 +231,35 @@ def test_user_log_as_argument(self):
         entries = ActivityLog.objects.for_user(user)
         assert len(entries) == 1
 
+    def test_ip_log(self):
+        addon = Addon.objects.get()
+        assert IPLog.objects.count() == 0
+        # Creating an activity log for an action without store_ip=True doesn't
+        # create an IPLog.
+        assert not getattr(amo.LOG.REJECT_VERSION, 'store_ip', False)
```

Comment:
Nit: You could put `amo.LOG.REJECT_VERSION` into a variable and use it both here and in the call to `ActivityLog.create` to guarantee they are the same. The same goes for `amo.LOG.ADD_VERSION` below.

Reply:
76a5f405685625eb007c9521981f1d7483755b2c